client/client_csi_endpoint_test.go

```diff
@@ -6,6 +6,7 @@ import (
 
 	"github.com/hashicorp/nomad/client/dynamicplugins"
 	"github.com/hashicorp/nomad/client/structs"
+	s "github.com/hashicorp/nomad/nomad/structs"
```

Comment:
could you use `nstructs` here to follow the conventions we have elsewhere in nomad? - It simplifies `git grep` and also reading comprehension

Reply:
done